packages/epinio/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,28 @@
+--- charts-original/Chart.yaml
++++ charts/Chart.yaml
+@@ -1,7 +1,15 @@
+ annotations:
+   artifacthub.io/license: Apache-2.0
++  catalog.cattle.io/certified: rancher
++  catalog.cattle.io/experimental: "true"
++  catalog.cattle.io/kube-version: '>= 1.20'
++  catalog.cattle.io/namespace: cattle-epinio-system
++  catalog.cattle.io/os: linux
++  catalog.cattle.io/rancher-version: '>= 2.6.0-0 <= 2.6.99-0'
++  catalog.cattle.io/release-name: epinio
++  catalog.cattle.io/type: cluster-tool
```

Comment:
Can you add `catalog.cattle.io/permits-os: linux,windows`? I saw you added the linux tolerations which go along with this annotation in order to allow deploying in an RKE2 hybrid cluster (linux and windows nodes). This will need to be tested in Windows clusters, to make sure pods aren't scheduled on Windows nodes.

Reply:
Done. Rebase, commit 4599b9feb20bd316169e303f21994961b55f2f74